test/built-ins/Proxy/get/trap-is-undefined-receiver.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2016 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.8
+description: >
```

Comment:
Generally we put raw spec under 'info' and use 'description' for a brief description of the test. Would you mind updating these?